### ISSUE TITLE
CUDPP CUDA 3.1 ERROR

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/cudpp#61]
```
What steps will reproduce the problem?
1. Update to cuda 3.1
2. Compile with cuda 3.1 nvcc
3.

What is the expected output? What do you see instead?
...
Compiling with CUDA Build Rule...
"C:\CUDA\bin\nvcc.exe" -m32 -gencode=arch=compute_20,code=sm_20 
-gencode=arch=compute_10,code=sm_10 -ccbin "C:\Program Files\Microsoft Visual 
Studio 9.0\VC\bin" -I"C:\CUDA\include" -I"./" -I"../../common/inc" 
-I"../../../shared/inc" -I"C:\CUDA\include" -I"./" -I"../common/inc" -I"./src" 
-I"./include" -Xcompiler "/EHsc /W3 /nologo /O2 /Zi /MT " -maxrregcount=32 
--compile -o "Release\segmented_scan_app.cu.obj" 
"c:\cudpp-1_1\cudpp_src_1.1.1\cudpp\src\app\segmented_scan_app.cu"
segmented_scan_app.cu
tmpxft_00000634_00000000-6_segmented_scan_app.compute_10.cudafe1.gpu
tmpxft_00000634_00000000-10_segmented_scan_app.compute_10.cudafe2.gpu
segmented_scan_app.cu
tmpxft_00000634_00000000-3_segmented_scan_app.compute_20.cudafe1.gpu
tmpxft_00000634_00000000-14_segmented_scan_app.compute_20.cudafe2.gpu
C:/Users/.../AppData/Local/Temp/tmpxft_00000634_00000000-15_segmented_scan_app.c
ompute_20.cpp3.i(0): Error: Writing intermediate compiler file 
(C:/Users/.../AppData/Local/Temp/ccBI#.a05072): Unknown error
Project : error PRJ0019: A tool returned an error code from "Compiling with 
CUDA Build Rule..."

What version of the product are you using? On what operating system?
CUDPP 1.1.1
Windows 7 32

Please provide any additional information below.
I used CUDA 3.0 and CUDPP 1.1.1 and its work fine. After installing of CUDA 
3.1, it was not possible to compile CUDPP.
I comment out  the lines 987-1112 of the "warpSegScan"-function and the 
compilation was succesful. But why?
```

Original issue reported on code.google.com by `derz...@online.de` on 14 Jul 2010 at 10:54

**Comment by GoogleCodeExporter** [comment]
```
A work around is compile with CUDA 3.0 and use with CUDA 3.1, but it is a bad 
solution... 
```

Original comment by `derz...@online.de` on 15 Jul 2010 at 5:37

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Have you got cudpp to compile with CUDA 3.2?

-Ritesh
```

Original comment by `rites...@gmail.com` on 31 Jan 2011 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
Yes, it work with CUDA 3.2 without problems...
```

Original comment by `derz...@online.de` on 31 Jan 2011 at 10:27

**Comment by GoogleCodeExporter** [comment]
Original comment by `rites...@gmail.com` on 1 Feb 2011 at 6:02
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Since this was not a CUDPP bug, I'm adding this comment to indicate that we 
didn't fix it -- the issue with CUDA 3.1 still exists.  I am changing the 
status to WontFix since we don't have time to fix the 3.1 issues.
```

Original comment by `harr...@gmail.com` on 1 Feb 2011 at 11:14
- Changed state: **WontFix**
